From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Meyer <katexochen0@gmail.com>
Date: Tue, 11 Nov 2025 11:04:23 +0100
Subject: [PATCH] genpolicy: remove non-existing field from CronJobSpec

There is no backoffLimit on CronJobSpec, also no additional fields.

Signed-off-by: Paul Meyer <katexochen0@gmail.com>
---
 src/tools/genpolicy/src/cronjob.rs | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/tools/genpolicy/src/cronjob.rs b/src/tools/genpolicy/src/cronjob.rs
index 0ed0e02dade22bd2ca50b9c2f66da201b084a947..d635c864606cc52017e1228aee9b3b53bac3ced6 100644
--- a/src/tools/genpolicy/src/cronjob.rs
+++ b/src/tools/genpolicy/src/cronjob.rs
@@ -54,10 +54,6 @@ pub struct CronJobSpec {
 
     #[serde(skip_serializing_if = "Option::is_none")]
     timeZone: Option<String>,
-
-    #[serde(skip_serializing_if = "Option::is_none")]
-    backoffLimit: Option<i32>,
-    // TODO: additional fields.
 }
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
